models/intel/person-detection-0106/description/person-detection-0106.md

```diff
@@ -0,0 +1,55 @@
+# person-detection-0106
+
+## Use Case and High-Level Description
+
+This is a person detector that is based on Cascade R-CNN architecture with ResNet50
+backbone.
+
+## Example
+
+![](./person-detection-0106.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| AP                              | 94.54% (internal test set)                |
+| Pose coverage                   | Standing upright, parallel to image plane |
+| Support of occluded persons     | YES                                       |
+| Occlusion coverage              | <50%                                      |
+| Min person height               | 100 pixels (on 1080p)                     |
+| GFlops                          | 404.264                                   |
+| MParams                         | 71.565                                    |
+| Source framework                | PyTorch\*                                 |
+
+Average Precision (AP) is defined as an area under
+the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall)
+curve. Intersection over union threshold of 0.5 is used for matching.
+
+## Performance
+
+## Inputs
+
+Name: `input`, shape: [1x3x800x1344] - An input image in the format [BxCxHxW],
+   where:
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
```

Comment:
Please, remove indentation

Reply:
Removed here and in readme files for some other models as well. Please take a look.